File: tests/data-clumps/test-cases/field-field/always-class/negative/typescript/source/Patient.ts
import { Address } from './Address';
import { ContactInfo } from './ContactInfo';
import { EmergencyContact } from './EmergencyContact';

export class Patient {
  public address: Address;
  public contactInfo: ContactInfo;
  public emergencyContact: EmergencyContact;
  public patientId: string;

  constructor(
    address: Address,
    contactInfo: ContactInfo,
    emergencyContact: EmergencyContact,
    patientId: string
  ) {
    this.address = address;
    this.contactInfo = contactInfo;
    this.emergencyContact = emergencyContact;
    this.patientId = patientId;
  }
}

File: tests/data-clumps/test-cases/field-field/always-class/negative/java/source/Patient.java
public class Patient {
    public Address address;
    public ContactInfo contactInfo;
    public EmergencyContact emergencyContact;
    public String patientId;

    public Patient(Address address, ContactInfo contactInfo, EmergencyContact emergencyContact, String patientId) {
        this.address = address;
        this.contactInfo = contactInfo;
        this.emergencyContact = emergencyContact;
        this.patientId = patientId;
    }
}
